frontend/src/protogen/redpanda/api/dataplane/v1alpha2/pipeline_pb.ts

```diff
@@ -653,19 +637,12 @@ export class PipelineUpdate extends Message<PipelineUpdate> {
   configYaml = "";
 
   /**
-   * Pipeline resource requests.
+   * The amount of resources that is guaranteed to be assigned to the pipeline.
```

Comment:
I'm ok with the single number describing the resources assigned to a pipeline in the public facing API, although we were already [tweaking request/limits in serverless](https://github.com/redpanda-data/serverless/blob/b1e5e138cce0b70012912081d90dac88f31121ca/infra/terraform/provisioners/kubernetes-proxy/modules/kubernetes-serverless-proxy-aws/files/console.yaml.tftpl#L47) to tune the over-subscription levels.

I'd be more in favor to remove the k8s naming and semantics from this field (req/limit) and just use a generic `resources` field. The reason is that in serverless, users will always see a certain amount of resources in the UI e.g. `100 millicores`, but under the hood we may decide to set a "k8s request" e.g. of `10 millicores` if CPU is mostly unused on average on some nodes.

So, TLDR the value the user configures in serverless is basically the "limit", not the "request" (which is on us to tune). Hence, better calling it just "resources"..